packages/react-hooks/helpers/requiredMembersThresholds.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright 2017-2021 @polkadot/react-hooks authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+export function getMoreThanThresholdMembersCount (membersCount: number, thresholdRatio: number): number {
+  if (membersCount === 0) { return 0; }
+
+  return Math.floor(membersCount * thresholdRatio) + 1;
+}
+
+export function getAtLeastThresholdMembersCount (membersCount: number, thresholdRatio: number): number {
+  return Math.ceil(membersCount * thresholdRatio);
+}
```

Comment:
Not sure why this is a helper? It can just be in the hook itself?

Reply:
It was extracted to enable simple testing

Reply:
Can still be exported from the hook and have the test for that? The `index.ts` generally doesn't do a `export *`